pkg/template/app/app.go

```diff
@@ -17,7 +17,7 @@ type Config struct {
 }
 
 type SecretConfig struct {
-	Data      map[string][]byte
+	Data      []byte
```

Comment:
Aligning this with how we template configmaps. The only difference is secrets use a `[]byte` type instead of a string.